lib/mongos.js

```diff
@@ -102,8 +102,8 @@ class Mongos extends EventEmitter {
       var opt = clone(self.clientOptions);
       opt.host = self.options.bind_ip;
       opt.port = self.options.port;
-      opt.connectionTimeout = 5000;
-      opt.socketTimeout = 5000;
+      opt.connectionTimeout = options.connectionTimeout || 5000;
```

Comment:
would you mind renaming these `connectionTimeoutMS` and `socketTimeoutMS`? Also, I know this is not on you _at all_, but could you add basic jsdocs above describing what these options do? Ideally in the future we would document everything for our future users